src/helpers/pairs.ts

```diff
@@ -149,3 +149,12 @@ export function getInterpolatorForPair({ from, to }: CSSPairs) {
     },
   };
 }
+
+export function getInterpolatorsForPairs({ from, to }: CSSPairs) {
+  return Object.keys(from).map(key => {
+    const f = { [key]: (from as { [cssProperty: string]: any })[key] };
+    const t = { [key]: (to as { [cssProperty: string]: any })[key] };
+
+    return getInterpolatorForPair({ from: f, to: t });
+  });
+}
```

Comment:
This allows for simultaneous interpolation of multiple CSS properties! For example:

```typescript
from: {
   opacity: 0,
   transform: 'scale(0)'
},
to: {
  opacity: 1,
  transform: 'scale(1)'
}
```